Doc/library/sqlite3.rst

```diff
@@ -1431,13 +1431,27 @@ case-insensitively by name:
 .. literalinclude:: ../includes/sqlite3/rowclass.py
 
 
+.. _sqlite3-connection-context-manager:
+
 Using the connection as a context manager
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Connection objects can be used as context managers
-that automatically commit or rollback transactions.  In the event of an
-exception, the transaction is rolled back; otherwise, the transaction is
-committed:
+A :class:`Connection` object can be used as a context manager that
+automatically commit or rollback open transactions when leaving the body of the
```

Comment:
```suggestion
automatically commits or rollbacks open transactions when leaving the body of the
```
I think the 's' is needed since object is singular

Reply:
Subtlety here... The noun for the action is a 'rollback', but the verb for it is 'to roll back' (two words). So, this should be 'automaticalky commits or rolls back'.

Reply:
```suggestion
automatically commits or rolls back open transactions when leaving the body of the
```